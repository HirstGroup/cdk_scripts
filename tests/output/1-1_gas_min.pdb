ATOM      1  C   UNL     1       6.514  -2.939   0.813  1.00  0.00           C  
ATOM      2  C1  UNL     1       5.040  -2.737   0.699  1.00  0.00           C  
ATOM      3  C2  UNL     1       4.172  -3.788   0.871  1.00  0.00           C  
ATOM      4  C3  UNL     1       2.806  -3.639   0.772  1.00  0.00           C  
ATOM      5  C4  UNL     1       2.268  -2.394   0.491  1.00  0.00           C  
ATOM      6  C5  UNL     1       0.823  -2.266   0.397  1.00  0.00           C  
ATOM      7  O   UNL     1       0.141  -3.229   0.817  1.00  0.00           O  
ATOM      8  N   UNL     1       0.179  -1.134  -0.133  1.00  0.00           N  
ATOM      9  C6  UNL     1      -1.197  -0.898  -0.272  1.00  0.00           C  
ATOM     10  C7  UNL     1      -2.209  -1.734   0.137  1.00  0.00           C  
ATOM     11  C8  UNL     1      -3.562  -1.445  -0.034  1.00  0.00           C  
ATOM     12  C9  UNL     1      -3.907  -0.271  -0.638  1.00  0.00           C  
ATOM     13  C10 UNL     1      -5.362   0.063  -0.864  1.00  0.00           C  
ATOM     14  N1  UNL     1      -6.207  -0.969  -0.290  1.00  0.00           N  
ATOM     15  C11 UNL     1      -7.605  -0.736  -0.650  1.00  0.00           C  
ATOM     16  C12 UNL     1      -8.005   0.715  -0.491  1.00  0.00           C  
ATOM     17  N2  UNL     1      -7.525   1.099   0.823  1.00  0.00           N  
ATOM     18  C13 UNL     1      -8.279   2.191   1.386  1.00  0.00           C  
ATOM     19  C14 UNL     1      -7.361  -0.019   1.691  1.00  0.00           C  
ATOM     20  C15 UNL     1      -6.181  -0.850   1.183  1.00  0.00           C  
ATOM     21  C16 UNL     1      -2.936   0.606  -1.069  1.00  0.00           C  
ATOM     22  C17 UNL     1      -3.320   1.883  -1.727  1.00  0.00           C  
ATOM     23  F   UNL     1      -2.160   2.580  -2.066  1.00  0.00           F  
ATOM     24  F1  UNL     1      -3.999   2.730  -0.846  1.00  0.00           F  
ATOM     25  F2  UNL     1      -4.013   1.668  -2.880  1.00  0.00           F  
ATOM     26  C18 UNL     1      -1.620   0.292  -0.886  1.00  0.00           C  
ATOM     27  C19 UNL     1       3.154  -1.354   0.323  1.00  0.00           C  
ATOM     28  C20 UNL     1       4.534  -1.493   0.420  1.00  0.00           C  
ATOM     29  C21 UNL     1       5.383  -0.349   0.229  1.00  0.00           C  
ATOM     30  C22 UNL     1       6.078   0.622   0.060  1.00  0.00           C  
ATOM     31  C23 UNL     1       6.885   1.792  -0.148  1.00  0.00           C  
ATOM     32  C24 UNL     1       6.374   3.013  -0.548  1.00  0.00           C  
ATOM     33  N3  UNL     1       7.403   3.871  -0.641  1.00  0.00           N  
ATOM     34  C25 UNL     1       8.560   3.240  -0.314  1.00  0.00           C  
ATOM     35  C26 UNL     1       9.872   3.683  -0.257  1.00  0.00           C  
ATOM     36  C27 UNL     1      10.833   2.769   0.120  1.00  0.00           C  
ATOM     37  C28 UNL     1      10.415   1.487   0.411  1.00  0.00           C  
ATOM     38  N4  UNL     1       9.135   1.086   0.346  1.00  0.00           N  
ATOM     39  N5  UNL     1       8.203   1.977  -0.020  1.00  0.00           N  
ATOM     40  H   UNL     1       6.739  -3.962   1.034  1.00  0.00           H  
ATOM     41  H1  UNL     1       6.914  -2.318   1.599  1.00  0.00           H  
ATOM     42  H2  UNL     1       6.999  -2.668  -0.111  1.00  0.00           H  
ATOM     43  H3  UNL     1       4.556  -4.722   1.081  1.00  0.00           H  
ATOM     44  H4  UNL     1       2.186  -4.452   0.908  1.00  0.00           H  
ATOM     45  H5  UNL     1       0.770  -0.412  -0.448  1.00  0.00           H  
ATOM     46  H6  UNL     1      -1.953  -2.623   0.595  1.00  0.00           H  
ATOM     47  H7  UNL     1      -4.286  -2.105   0.290  1.00  0.00           H  
ATOM     48  H8  UNL     1      -5.550   0.130  -1.915  1.00  0.00           H  
ATOM     49  H9  UNL     1      -5.583   0.998  -0.393  1.00  0.00           H  
ATOM     50  H10 UNL     1      -5.869  -1.870  -0.627  1.00  0.00           H  
ATOM     51  H11 UNL     1      -8.227  -1.336  -0.019  1.00  0.00           H  
ATOM     52  H12 UNL     1      -7.726  -0.998  -1.681  1.00  0.00           H  
ATOM     53  H13 UNL     1      -9.069   0.820  -0.547  1.00  0.00           H  
ATOM     54  H14 UNL     1      -7.594   1.331  -1.263  1.00  0.00           H  
ATOM     55  H15 UNL     1      -6.583   1.472   0.703  1.00  0.00           H  
ATOM     56  H16 UNL     1      -9.291   1.881   1.545  1.00  0.00           H  
ATOM     57  H17 UNL     1      -8.262   3.021   0.711  1.00  0.00           H  
ATOM     58  H18 UNL     1      -7.844   2.481   2.320  1.00  0.00           H  
ATOM     59  H19 UNL     1      -8.250  -0.614   1.688  1.00  0.00           H  
ATOM     60  H20 UNL     1      -7.175   0.313   2.691  1.00  0.00           H  
ATOM     61  H21 UNL     1      -6.231  -1.828   1.613  1.00  0.00           H  
ATOM     62  H22 UNL     1      -5.274  -0.359   1.468  1.00  0.00           H  
ATOM     63  H23 UNL     1      -0.904   0.960  -1.215  1.00  0.00           H  
ATOM     64  H24 UNL     1       2.772  -0.419   0.113  1.00  0.00           H  
ATOM     65  H25 UNL     1       5.382   3.226  -0.739  1.00  0.00           H  
ATOM     66  H26 UNL     1      10.120   4.659  -0.487  1.00  0.00           H  
ATOM     67  H27 UNL     1      11.828   3.034   0.182  1.00  0.00           H  
ATOM     68  H28 UNL     1      11.127   0.796   0.694  1.00  0.00           H  
TER      69      UNL     1 
END   
