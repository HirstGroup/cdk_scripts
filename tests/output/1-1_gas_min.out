
          -------------------------------------------------------
          Amber 21 SANDER                              2021
          -------------------------------------------------------

| Run on 06/21/2023 at 17:59:58

|   Executable path: sander
| Working directory: /home/pczbf/cdk_scripts/tests/output
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: min.in                                                                
| MDOUT: 1-1_gas_min.out                                                       
|INPCRD: 1-1_gas.rst7                                                          
|  PARM: 1-1_gas.parm7                                                         
|RESTRT: 1-1_gas_min.rst7                                                      
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: mdcrd                                                                 
|MDINFO: mdinfo                                                                
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

Minimization                                                                   
 &cntrl                                                                        
   bellymask='@1'                                                              
   cut=999,                                                                    
   drms=0.001,                                                                 
   DT=0.002,                                                                   
   ibelly=1,                                                                   
   ig=-1,                                                                      
   igb=5,                                                                      
   imin=1,                                                                     
   ioutfm=1,                                                                   
   irest=0,                                                                    
   maxcyc=5000,                                                                
   ncyc=5000,                                                                  
   NSTLIM=5000,                                                                
   ntb=0,                                                                      
   ntmin=1,                                                                    
   ntp=0,                                                                      
   ntpr=100,                                                                   
   NTR=0,                                                                      
   ntwe=0,                                                                     
   ntwr=5000,                                                                  
   ntwx=100,                                                                   
   ntx=1,                                                                      
 &end                                                                          

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 06/21/23 Time = 16:53:06
 NATOM  =      68 NTYPES =      11 NBONH =      29 MBONA  =      43
 NTHETH =      65 MTHETA =      62 NPHIH =     120 MPHIA  =      91
 NHPARM =       0 NPARM  =       0 NNB   =     379 NRES   =       1
 NBONA  =      43 NTHETA =      62 NPHIA =      91 NUMBND =      28
 NUMANG =      48 NPTRA  =      15 NATYP =      19 NPHB   =       0
 IFBOX  =       0 NMXRS  =      68 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are H(N)-modified Bondi radii (mbondi2)                                             

|     Memory Use     Allocated
|     Real              708384
|     Hollerith            207
|     Integer            23489
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total             5626 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to    45548 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

*****                                                                           

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =     100, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     100, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       5, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.80000, gbgamma =   4.85000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       1, ntr     =       0

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00100

    LOADING THE BELLY ATOMS AS GROUPS

     Mask @1; matches     1 atoms
|  INFO: Old style inpcrd file read


--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

*****                                                                           
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       2.1116E+00     1.2067E+01     2.0582E+01     C           1

 BOND    =        0.7257  ANGLE   =        0.1796  DIHED      =        0.0000
 VDWAALS =       -0.4532  EEL     =       -2.0875  EGB        =        1.9616
 1-4 VDW =        1.8049  1-4 EEL =       -0.0195  RESTRAINT  =        0.0000


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     42       1.9344E+00     5.4072E-04     9.1695E-04     C           1

 BOND    =        0.6324  ANGLE   =        0.0523  DIHED      =        0.0000
 VDWAALS =       -0.4539  EEL     =       -2.0917  EGB        =        1.9636
 1-4 VDW =        1.8512  1-4 EEL =       -0.0195  RESTRAINT  =        0.0000

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 3.33% of Total)
|    Fast Water setup           0.00 ( 0.05% of Total)
|                Calc gb radii              0.00 (93.53% of Gen B)
|                Calc gb diag               0.00 ( 2.30% of Gen B)
|                Calc gb off-diag           0.00 ( 3.83% of Gen B)
|                Other                      0.00 ( 0.34% of Gen B)
|             Gen Born time              0.00 (100.0% of Nonbo)
|             IPS excludes               0.00 ( 0.05% of Nonbo)
|          Nonbond force              0.00 (93.99% of Force)
|          Bond/Angle/Dihedral        0.00 ( 4.64% of Force)
|          Other                      0.00 ( 1.37% of Force)
|       Force time                 0.00 (100.0% of Runmd)
|    Runmd Time                 0.00 (36.48% of Total)
|    Other                      0.01 (60.13% of Total)
| Total time                 0.01 (98.99% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 17:59:58.040  on 06/21/2023
|           Setup done at 17:59:58.047  on 06/21/2023
|           Run   done at 17:59:58.052  on 06/21/2023
|     wallclock() was called     784 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
